R/quantifyTranscriptionFactorSimilarity.R

```diff
@@ -0,0 +1,152 @@
+# quantifyTranscriptionFactorSimilarity.R
+#
+# Purpose: To quantify similarity between TF co-occurence and absence
+# Version: 1.0
+# Date: 2019-03-25
+# Author: Rachel Woo
+# License: MIT
+#
+# Input: An excel spreadsheet of the system parsed using the BCB420-2019-resources scripts.
+# Output: Dengrogram of TF  co-occurance written to
+#         ~/inst/img/TFOccuranceDendrogram.jpg
+# Dependencies: xlsx, tidyverse, readxl
+# ==============================================================================
+
+# SIDE EFFECTS:
+# Displays dendrogram
+
+# # ====  PACKAGES  ==============================================================
+# # Get packages
+# if (requireNamespace("xlsx", quietly=TRUE)) {
+#   install.packages("tidyverse", repos = "http://cran.us.r-project.org")
+# }
+# if (requireNamespace("devtools", quietly=TRUE)) {
+#   install.packages("devtools", repos = "http://cran.us.r-project.org")
+#   install.packages("Biostrings", repos = "http://cran.us.r-project.org")
+#   install.packages("Rtools", repos = "http://cran.us.r-project.org")
+# }
+#
+#
+# # Load packages
+# require(readxl, quietly = TRUE)
+
+
+# ==== FUNCTIONS ===============================================================
+
+
+#' Helper Function: Read system data from excel
```

Comment:
For every function, you also need to add the function name.
Please check this link for the template of roxigen headers
https://github.com/hyginn/BCB420.2019.ESA/blob/master/dev/functionTemplate.R